docker/nginx/conf.d/include/ssl-settings

```diff
@@ -0,0 +1,13 @@
+# https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=intermediate&openssl=1.1.1d&hsts=false&ocsp=false&guideline=5.6
+
+ssl_session_timeout 1d;
```

Comment:
seems high

Reply:
that's what mozilla suggests in https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=intermediate&openssl=1.1.1d&hsts=false&ocsp=false&guideline=5.6 so I'm just hoping they know a bit more about certificates than I do :)